tools/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
 // +build tools
```

Comment:
Should we retain the old style build tags, since we have bumped minimum supported version in mod file to 1.17?

Reply:
let's keep for now, we will remove later

Reply:
I forgot to mention this earlier. Build tags are the first line in any Golang file. So It's effectively a comment right now.